style.css

```diff
@@ -0,0 +1,226 @@
+html {
```

Comment:
missing * { box-sizing: border-box; } from the top of the file to correctly set the box sizing for the document